### ISSUE TITLE
Feature request (or is it just a bug?): Refresh (re-read) thumbnails from all books in bookshelf

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#243]
```
Bookshelf has brilliant feature to scan particular folder. But all new books 
have blank thumbnails. Is it possible to add menu item to re-read all 
thumbnails from all books on bookshelf? That will make it pretty right from the 
start.

```

Original issue reported on code.google.com by `dbara...@gmail.com` on 19 Jun 2012 at 2:31

**Comment by GoogleCodeExporter** [comment]
```
Ну вообще то изготовление превьюшек 
процесс достаточно трудоемкий.
Недаром они делаются уже из отрисованной 
страницы.
Отдельно может и сделаем когда нибудь.
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 19 Jun 2012 at 2:43
- Changed state: **Postponed**
- Added labels: **Priority-Low**, **Type-Enhancement**, **thumbnails**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Спасибо, что не забудете об этом предожении.
Его можно сделать не автоматическим, а 
user-callable, чтобы он сам мог это сделать, если 
ему надо...
```

Original comment by `mark.uly...@gmail.com` on 17 Oct 2012 at 4:20

**Comment by GoogleCodeExporter** [comment]
```
Мы помним. Честно. Если не хотим делать, 
сразу ставим wontfix. Если поставили postponed то 
мы вернемся к этому. Просто поймите - 
фактически программистов сейчас двое и это 
не основная работа. Только в свободное 
время. Разобраться бы с горой багов на всем 
зоопарке устройств, потом уж улучшения и 
доработки.
```

Original comment by `Andrei.K...@gmail.com` on 17 Oct 2012 at 8:57

**Comment by GoogleCodeExporter** [comment]
```
Implemented in version 2.1
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 25 Dec 2013 at 10:03
- Changed state: **Fixed**
